torchbenchmark/models/attention_is_all_you_need_pytorch/__init__.py

```diff
@@ -7,8 +7,14 @@
 import torch
 import torch.nn.functional as F
 import torch.optim as optim
-from torchtext.data import Field, Dataset, BucketIterator
-from torchtext.datasets import TranslationDataset
+import torchtext.data
+from torchtext.data import Dataset, BucketIterator
+if torchtext.__version__ >= "0.9":
```

Comment:
does it work to say 'if torchtext.__version__' 
without first importing torchtext?

Reply:
Tried it, it doesn't work.